experimental/table/server/master/src/main/java/alluxio/master/catalog/AlluxioCatalog.java

```diff
@@ -139,7 +144,7 @@ public Table getTable(String dbName, String tableName) throws IOException {
   public List<String> getAllTables(String dbName) throws IOException {
     Database db = mDBs.get(dbName);
     if (db == null) {
-      throw new IOException(String.format("Database %s does not exist", dbName));
+      throw new NotFoundException(String.format("Database %s does not exist", dbName));
```

Comment:
Consider making a utility method for getting databases and tables by name if we need to do this logic often.

Reply:
done.